src/boilerplates/create_wallet_linking_embedded_wallet_react.ts

```diff
@@ -0,0 +1,56 @@
+
+import { Command } from "commander";
+import { promptForGoogleClientIdWithLogs, promptForProjectAccessKeyWithLogs, promptForWaaSConfigKeyWithLogs, promptForWalletConnectIdWithLogs, promptUserKeyCustomizationDecision, writeDefaultKeysToEnvFileIfMissing, writeToEnvFile } from "../utils";
+import { EnvKeys } from "../utils/types";
+
+import shell from "shelljs";
+
+const WALLET_LINKING_EMBEDDED_WALLET_REACT_REPO_URL = "https://github.com/0xsequence-demos/demo-embedded-wallet-linking";
+
+export async function createWalletLinkingEmbeddedWalletReact(program: Command, options: any) {
+    let waasConfigKey = options.waasConfigKey;
+    let projectAccessKey = options.projectAccessKey;
+    let googleClientId = options.googleClientId;
+    let walletConnectId = options.walletConnectId;
+
+    const userWantsToConfigureTheirKeys = await promptUserKeyCustomizationDecision();
+    
+    if (userWantsToConfigureTheirKeys) {
+        waasConfigKey = await promptForWaaSConfigKeyWithLogs(waasConfigKey);
+        projectAccessKey = await promptForProjectAccessKeyWithLogs(projectAccessKey);
+        googleClientId = await promptForGoogleClientIdWithLogs(googleClientId);
+        walletConnectId = await promptForWalletConnectIdWithLogs(walletConnectId);
+    }
+
+    console.log("Cloning the repo to `demo-embedded-wallet-linking`...");
+
+    shell.exec(`git clone ${WALLET_LINKING_EMBEDDED_WALLET_REACT_REPO_URL} demo-embedded-wallet-linking`, { silent: !options.verbose });
```

Comment:
Most repositories follow the -boilerplate pattern when creating repository files. Why would this one be any different?

Reply:
Some examples:
* `git clone ${EMBEDDED_WALLET_NEXTJS_REPO_URL} kit-embedded-wallet-nextjs-boilerplate`
* `git clone ${TX_MANAGER_REPO_URL} server-side-transactions-boilerplate`
* `git clone ${EMBEDDED_WALLET_REACT_REPO_URL} embedded-wallet-react-boilerplate`
* `git clone ${EMBEDDED_WALLET_VERIFY_SESSION_REPO_URL} embedded-wallet-verify-session-boilerplate`

Reply:
this is more of a demo, not necessarily a boilerplate, but people can still use it